Test `TestRedshiftUnloadTypeMapping.testVarbinary` is failing

https://github.com/trinodb/trino/actions/runs/19572217039/job/56048341054?pr=27388

```
Error:  Tests run: 26, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 560.9 s <<< FAILURE! -- in io.trino.plugin.redshift.TestRedshiftUnloadTypeMapping
Error:  io.trino.plugin.redshift.TestRedshiftUnloadTypeMapping.testVarbinary -- Time elapsed: 3.769 s <<< ERROR!
io.trino.testing.QueryFailedException: Table 'test_schema.test_varbinary2uxztefsvf' has no supported columns (all 19 columns are not supported)
	at io.trino.testing.AbstractTestingTrinoClient.execute(AbstractTestingTrinoClient.java:138)
	at io.trino.testing.DistributedQueryRunner.executeInternal(DistributedQueryRunner.java:587)
	at io.trino.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:570)
	at io.trino.sql.query.QueryAssertions$QueryAssert.lambda$new$1(QueryAssertions.java:317)
	at com.google.common.base.Suppliers$NonSerializableMemoizingSupplier.get(Suppliers.java:201)
	at io.trino.sql.query.QueryAssertions$QueryAssert.result(QueryAssertions.java:436)
	at io.trino.testing.datatype.SqlDataTypeTest.verifySelect(SqlDataTypeTest.java:102)
	at io.trino.testing.datatype.SqlDataTypeTest.execute(SqlDataTypeTest.java:90)
	at io.trino.testing.datatype.SqlDataTypeTest.execute(SqlDataTypeTest.java:83)
	at io.trino.plugin.redshift.TestRedshiftTypeMapping.testVarbinary(TestRedshiftTypeMapping.java:286)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:511)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.tryRemoveAndExec(ForkJoinPool.java:1490)
	at java.base/java.util.concurrent.ForkJoinPool.helpJoin(ForkJoinPool.java:2248)
	at java.base/java.util.concurrent.ForkJoinTask.awaitDone(ForkJoinTask.java:499)
	at java.base/java.util.concurrent.ForkJoinTask.join(ForkJoinTask.java:666)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:511)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1450)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2019)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187)
	Suppressed: java.lang.Exception: SQL: SELECT * FROM test_schema.test_varbinary2uxztefsvf
		at io.trino.testing.DistributedQueryRunner.executeInternal(DistributedQueryRunner.java:594)
		... 18 more
Caused by: io.trino.spi.connector.TableNotFoundException: Table 'test_schema.test_varbinary2uxztefsvf' has no supported columns (all 19 columns are not supported)
	at io.trino.plugin.jdbc.BaseJdbcClient.getColumns(BaseJdbcClient.java:360)
	at io.trino.plugin.jdbc.ForwardingJdbcClient.getColumns(ForwardingJdbcClient.java:111)
	at io.trino.plugin.jdbc.jmx.StatisticsAwareJdbcClient.lambda$getColumns$0(StatisticsAwareJdbcClient.java:133)
	at io.trino.plugin.jdbc.jmx.JdbcApiStats.wrap(JdbcApiStats.java:34)
	at io.trino.plugin.jdbc.jmx.StatisticsAwareJdbcClient.getColumns(StatisticsAwareJdbcClient.java:133)
	at io.trino.plugin.jdbc.RetryingJdbcClient.lambda$getColumns$0(RetryingJdbcClient.java:108)
	at dev.failsafe.Functions.lambda$toCtxSupplier$11(Functions.java:243)
	at dev.failsafe.Functions.lambda$get$0(Functions.java:46)
	at dev.failsafe.internal.RetryPolicyExecutor.lambda$apply$0(RetryPolicyExecutor.java:74)
	at dev.failsafe.SyncExecutionImpl.executeSync(SyncExecutionImpl.java:187)
	at dev.failsafe.FailsafeExecutor.call(FailsafeExecutor.java:376)
	at dev.failsafe.FailsafeExecutor.get(FailsafeExecutor.java:112)
	at io.trino.plugin.jdbc.RetryingModule.retry(RetryingModule.java:67)
	at io.trino.plugin.jdbc.RetryingJdbcClient.getColumns(RetryingJdbcClient.java:108)
	at io.trino.plugin.jdbc.CachingJdbcClient.lambda$getColumns$0(CachingJdbcClient.java:185)
	at io.trino.cache.EmptyCache.get(EmptyCache.java:95)
	at io.trino.plugin.jdbc.CachingJdbcClient.get(CachingJdbcClient.java:834)
	at io.trino.plugin.jdbc.CachingJdbcClient.getColumns(CachingJdbcClient.java:185)
	at io.trino.plugin.jdbc.CachingJdbcClient.lambda$getColumns$0(CachingJdbcClient.java:185)
	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4859)
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3556)
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2307)
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2180)
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2071)
	at com.google.common.cache.LocalCache.get(LocalCache.java:3985)
	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4854)
	at io.trino.cache.EvictableCache.get(EvictableCache.java:119)
	at io.trino.plugin.jdbc.CachingJdbcClient.get(CachingJdbcClient.java:834)
	at io.trino.plugin.jdbc.CachingJdbcClient.getColumns(CachingJdbcClient.java:185)
	at io.trino.plugin.jdbc.DefaultJdbcMetadata.getTableSchema(DefaultJdbcMetadata.java:1045)
	at io.trino.plugin.base.classloader.ClassLoaderSafeConnectorMetadata.getTableSchema(ClassLoaderSafeConnectorMetadata.java:274)
	at io.trino.tracing.TracingConnectorMetadata.getTableSchema(TracingConnectorMetadata.java:234)
	at io.trino.metadata.MetadataManager.getTableSchema(MetadataManager.java:486)
	at io.trino.tracing.TracingMetadata.getTableSchema(TracingMetadata.java:310)
	at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitTable(StatementAnalyzer.java:2336)
	at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitTable(StatementAnalyzer.java:533)
	at io.trino.sql.tree.Table.accept(Table.java:70)
	at io.trino.sql.tree.AstVisitor.process(AstVisitor.java:27)
	at io.trino.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:552)
	at io.trino.sql.analyzer.StatementAnalyzer$Visitor.analyzeFrom(StatementAnalyzer.java:5099)
	at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitQuerySpecification(StatementAnalyzer.java:3156)
	at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitQuerySpecification(StatementAnalyzer.java:533)
	at io.trino.sql.tree.QuerySpecification.accept(QuerySpecification.java:155)
	at io.trino.sql.tree.AstVisitor.process(AstVisitor.java:27)
	at io.trino.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:552)
	at io.trino.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:560)
	at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitQuery(StatementAnalyzer.java:1601)
	at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitQuery(StatementAnalyzer.java:533)
	at io.trino.sql.tree.Query.accept(Query.java:130)
	at io.trino.sql.tree.AstVisitor.process(AstVisitor.java:27)
	at io.trino.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:552)
	at io.trino.sql.analyzer.StatementAnalyzer.analyze(StatementAnalyzer.java:512)
	at io.trino.sql.analyzer.StatementAnalyzer.analyze(StatementAnalyzer.java:501)
	at io.trino.sql.analyzer.Analyzer.analyze(Analyzer.java:98)
	at io.trino.sql.analyzer.Analyzer.analyze(Analyzer.java:87)
	at io.trino.execution.SqlQueryExecution.analyze(SqlQueryExecution.java:283)
	at io.trino.execution.SqlQueryExecution.<init>(SqlQueryExecution.java:218)
	at io.trino.execution.SqlQueryExecution$SqlQueryExecutionFactory.createQueryExecution(SqlQueryExecution.java:883)
	at io.trino.dispatcher.LocalDispatchQueryFactory.lambda$createDispatchQuery$0(LocalDispatchQueryFactory.java:158)
	at io.trino.$gen.Trino_testversion____20251121_134413_1.call(Unknown Source)
	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:128)
	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:74)
	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:80)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614)
	at java.base/java.lang.Thread.run(Thread.java:1474)
```

--- a/plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/RedshiftClient.java
+++ b/plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/RedshiftClient.java
@@ -625,6 +625,12 @@ public class RedshiftClient
                     RedshiftClient::readTime,
                     RedshiftClient::writeTime));
         }
+        if ("binary varying".equals(type.jdbcTypeName().orElse("")) || type.jdbcType() == Types.LONGVARBINARY) {
+            return Optional.of(ColumnMapping.sliceMapping(
+                    VARBINARY,
+                    varbinaryReadFunction(),
+                    varbinaryWriteFunction()));
+        }
 
         switch (type.jdbcType()) {
             case Types.BIT: // Redshift uses this for booleans
@@ -681,12 +687,6 @@ public class RedshiftClient
                                 : createUnboundedVarcharType(),
                         true));
             }
-
-            case Types.LONGVARBINARY:
-                return Optional.of(ColumnMapping.sliceMapping(
-                        VARBINARY,
-                        varbinaryReadFunction(),
-                        varbinaryWriteFunction()));
 
             case Types.DATE:
                 return Optional.of(ColumnMapping.longMapping(

